tests/python/gaia-ui-tests/gaiatest/apps/clock/regions/alarm.py

```diff
@@ -2,77 +2,63 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import time
-
 from marionette_driver import expected, By, Wait
 
 from gaiatest.apps.clock.app import Clock
+from gaiatest.apps.base import PageRegion
 
 
-class NewAlarm(Clock):
-
-    _alarm_name_locator = (By.ID, 'alarm-name')
-    _repeat_menu_locator = (By.ID, 'repeat-menu')
-    _sound_menu_locator = (By.ID, 'sound-menu')
-    _snooze_menu_locator = (By.ID, 'snooze-menu')
-    _done_locator = (By.ID, 'alarm-done')
-    _time_button_locator = (By.XPATH, "//li[input[@id='time-select']]")
+class Alarm(Clock):
```

Comment:
I don't think `Clock.__init__()` has special. The init method in Clock is the same as [Base](https://github.com/mozilla-b2g/gaia/blob/eb1193cd6d71e260744754c6fda6f67f2f90d90f/tests/python/gaia-ui-tests/gaiatest/apps/clock/app.py#L21). `PageRegion` also uses that same `__init__()`